params/Lukasz's_bird.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.1245
+    
+# e.g. "anas_krystallinus"    
+species.name <- " Lukasz's penguin "
```

Comment:
It would be great if you could remove the extra white space please.

```suggestion
species.name <- "Lukasz's penguin"
```